### ISSUE TITLE
Planned maneuver nodes are saved to the server but not loaded

### ISSUE THREAD

**Issue by 81ninja** [godarklight/DarkMultiPlayer#388]
Intentionally:

```
VesselWorker.cs:
private ProtoVessel CreateSafeProtoVesselFromConfigNode(ConfigNode inputNode, Guid protovesselID)
        {
            ProtoVessel pv = null;
            try
            {
                DodgeVesselActionGroups(inputNode);
                RemoveManeuverNodesFromProtoVessel(inputNode);
```

However I don't see the reason why this is. I've commented that line out and nobody died, nothing crashed, and maneuvers are correctly loaded. Does that have any side effects? Am I missing something?

**Comment by RockyTV** [comment]
I don't know why we remove them from the vessel, but if nothing broke if you commented that line, I think we're good to go to remove it. Let me know if hell breaks loose because of that.

**Comment by 81ninja** [comment]
Fixed on master by b9bc0d3
